snapshot/lib/snapshot/runner.rb

```diff
@@ -204,6 +207,21 @@ def erase_simulator(device_type)
       `xcrun simctl erase #{device_udid} &> /dev/null`
     end
 
+    def localize_simulator(device_type, language, locale)
+      apple_locale = language.sub("-", "_")
+      apple_languages = [language]
+
+      UI.message "Localizing #{device_type} (AppleLocale=#{apple_locale} AppleLanguages=[#{apple_languages}])"
```

Comment:
I think this output may double-up the [] around the language in the string output, is that intentional?